### PR TITLE
add MDAL formats info to README

### PR DESCRIPTION
**Pull request by PeterPetrik**
add MDAL info to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,13 @@ modified.
  * Oracle
  * WFS
 
+## Supported mesh formats include:
+
+ * NetCDF
+ * GRIB
+ * 2DM 
+ * [Other MDAL supported formats](https://github.com/lutraconsulting/MDAL#supported-formats)
```
Comment (luipir) [inline README.md]: the reason to point to an external documentation page when PDAL is already in core?

Comment (PeterPetrik) [reply README.md]: this is about MDAL, not PDAL? 

MDAL is build with QGIS  as "external" library (CMAKE flag WITH_INTERNAL_MDAL) only until it is stable enough to be treated similarly to GDAL or other external dependencies.

Comment (luipir) [reply README.md]: yes sorry... too used to talk about pdal.
Ok, Understood, tnx

Comment (m-kuhn) [reply README.md]: > MDAL is build with QGIS as "external" library (CMAKE flag WITH_INTERNAL_MDAL) only until it is stable enough to be treated similarly to GDAL or other external dependencies.

Out of curiosity, is there an ETA for this?

Comment (PeterPetrik) [reply README.md]: good question ... I think MDAL is is a good shape now, but this is definitely something that I want to open/discuss in Spain. I think it should be "internal" for this QGIS LTR, but maybe slowly time to go for regular distribution after 3.6 is out.

Comment (m-kuhn) [reply README.md]: Wow, glad to hear. A discussion in Spain will be appreciated.
However, I think starting to package MDAL for the most important platforms is something that can be started without any big discussion and distributions can gradually switch over where MDAL is available as standalone and based on the discussions by QGIS.

**Pull request merged**